src/pvecontrol/actions/node.py

```diff
@@ -27,18 +27,19 @@ def action_nodeevacuate(proxmox, args):
     targets = []
     # compute targets migration possible
     if args.target:
-        for t in list(set(args.target)):
-            if t == srcnode.node:
-                print(f"Target node {t} is the same as source node, skipping")
+        for pattern in list(set(args.target)):
+            if pattern == srcnode.node:
```

Comment:
We should move this check above the `find_nodes` in order to exclude source node when selection using pattern can match it. exemple:
 - source: pve-test-1
 - target: pve-test-*

 result curently:  ['pve-test-1', 'pve-test-2', 'pve-test-3']
 wanter result: ['pve-test-2','pve-test-3']